src/App.jsx

```diff
@@ -14,70 +15,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
```

Comment:
Consider it will be better
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```